src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+

```

Comment:
<img width="1013" alt="Screenshot 2022-01-17 at 08 58 18" src="https://user-images.githubusercontent.com/33026471/149722866-fd0b89d7-87ce-4d33-b2ee-9f6ee2922813.png">
Let's remove redundant empty lines here and in main